components/Dialogs/tests/unit/MDCAlertControllerTests.m

```diff
@@ -257,6 +291,20 @@ - (void)testAlertControllerSettingTitleAndMessage {
   XCTAssertEqual(view.messageLabel.text, message);
 }
 
+- (void)testAlertControllerSettingTitleAndAttributedMessage {
+  // Given
+  NSString *title = @"title";
+  NSString *message = @"attributed message";
+
+  // When
+  self.attributedAlert.titleFont = [UIFont systemFontOfSize:25];
```

Comment:
It looks like you're not actually setting the title + message values here.

Reply:
It's done in the setup of the test suite. This test is analogous to [testAlertControllerSettingTitleAndMessage](https://github.com/material-components/material-components-ios/pull/9937/files/c01c17ec9e445bae60a15934fc334caceb3fb882#diff-0db35bfda572f0ea27fbfde6ccdc8b31R280-R292) 😅